runtime/runtime/src/genesis.rs

```diff
@@ -32,7 +36,10 @@ impl<'a> StorageComputer<'a> {
         Self { result: HashMap::new(), config: &config.transaction_costs.storage_usage_config }
     }
 
+    /// Updates user's storage info based on the StateRecord.
     pub fn process_record(&mut self, record: &StateRecord) {
+        /// Note: It's okay to use unsafe math here, because this method should only be called on the trusted
+        /// state records (e.g. at launch from genesis)
```

Comment:
```suggestion
        // Note: It's okay to use unsafe math here, because this method should only be called on the trusted
        // state records (e.g. at launch from genesis)
```

`///` only works for definitions displayed by `cargo doc`. Three slashes is not a mere convention, it's a first-class syntax supported by the compiler for documentation comments on some, but not all, entities.